page.php

```diff
@@ -0,0 +1,42 @@
+<?php
+	get_header();
+	the_post();
+?>
+<section class="main-content">
+	<header class="page-header">
+		<div class="container">
+			<div class="columns is-centered">
+				<div class="column">
+					<?php
+					if ( function_exists( 'yoast_breadcrumb' ) ) {
+						yoast_breadcrumb( '<p class="breadcrumbs" id="breadcrumbs">', '</p>' );
+					}
+					?>
+					<h2 class="page-title"><?php echo CC_Site::page_title(); ?></h2>
+				</div>
+			</div>
+		</div>
+	</header>
+	<div class="container">
+		<div class="columns padding-vertical-larger">
+			<div class="column is-8">
+				<?php
+				if ( have_posts() ) :
+					while ( have_posts() ) :
+						the_post();
+						echo Components::simple_entry( get_the_ID(), true, true );
```

Comment:
why did you add a component here?
`simple_entry` is a new entry I use for a couple of widgets.
this one

![Screen Shot 2020-10-07 at 8 04 14 AM](https://user-images.githubusercontent.com/894708/95323282-4dd58700-0874-11eb-83f9-f109248fdfdd.png)